jstests/ssl/x509/mkcert.py

```diff
@@ -479,6 +482,35 @@ def process_client_multivalue_rdn(cert):
     os.remove(rsa)
     os.remove(pem)
 
+def convert_ecdsa_key(ec_key_file, pkcs8_key_file):
```

Comment:
Can we title this method letting people know that this is converting the key to a pkcs8 key?